src/utils/getUserById.tsx

```diff
@@ -0,0 +1,5 @@
+import usersFromServer from '../api/users';
```

Comment:
[CODE KNOWLEDGE] - You should prepare data in one place (App component) and pass it to child components. Avoid using 'import' for data in non-root components. Consider passing the necessary data as props to the components that need it.